website/docs/reference/resource-configs/docs.md

```diff
@@ -93,9 +139,7 @@ analyses:
 
 <TabItem value="macros">
 
-<!----
-To-do: check this
---->
+You can use the docs property in YAML files, _except_ in `dbt_project.yml`. Refer to [Macro properties](/reference/macro-properties) for more info.
```

Comment:
After this change, I'd suggest removing this line below so that we're not referring to it back-to-back:
```
Also refer to [macro properties](/reference/macro-properties).
```